wordpress-mobile/WordPress-iOS
# Tweak My Site Dashboard Cards UI

## Requirement

> Changing the font from serif NY -> sans serif SF, hiding the little title icons, hiding chevrons, adding hyperlinks - @osullivanchris 

Source pbArwn-5Uc-p2#comment-7395

## Screenshots

| WordPress | Jetpack |
| ----------- | -------- |
| ![](https://user-images.githubusercontent.com/9609223/224322539-9cf1482e-81ca-46f0-a2f3-e900f01861c8.png) | ![](https://user-images.githubusercontent.com/9609223/224321928-a4f070aa-1c72-4ab6-8947-6b2401b85e76.png) |

## Test Instructions

1. Install WordPress and Jetpack
2. Head to **My Site > Home**
3. **Verify** on both apps that the cards title icon and chevron are removed.
4. **Verify** on Jetpack that serif font is no longer used in the dashboard cards.

## Regression Notes
1. Potential unintended areas of impact
None.

5. What I did to test those areas of impact (or what existing automated tests I relied on)
None.

6. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:
